### DISCUSSION TITLE
Show folders instead of files

### DISCUSSION THREAD

**Discussion by Joopse** [croneter/PlexKodiConnect#1566] [Q&A] [answered]
Hi all, 

I'm very new to PKC, and I must say it works great for me except for one small thing (so far) Before this I used Kodi and Plex separately for different devices.

I have one folder with video files, which I don't want to keep for long, as a menu item. I had this in Kodi and it showed all the subfolders + individual files which weren't in a folder. With PKC it doesn't show my subfolders anymore. It just puts all files together whether they are in a subfolder or in the main folder.

Is it possible to make it show the subfolders again? I can't find an option in the Kodi settings or in the PKC settings (in Kodi) am I missing something?

I'm using the Amber skin by the way. (is there a consensus on the best skin to use with PKC + Kodi 19.1? I'm not sold on it yet.)

Thanks!

**Comment by croneter** [comment, marked as answer]
Check this: https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins

Also, try to navigate to Add-ons - Video Add-ons - PlexKodiConnect - [name of your library] - Browse by folder

**Reply by Joopse** [reply]
Thank you! I'll have a look. That last series of commands is when I add the folder to my Kodi home menu? If so, I completely missed that while setting up and would seem to be the solution.

**Reply by Joopse** [reply]
> Also, try to navigate to Add-ons - Video Add-ons - PlexKodiConnect - [name of your library] - Browse by folder

You're completely right! This is what I needed, but now there's another problem, I can't choose how to sort the list. It sorts by title, but there are no other options. I need it to be on "date added". Is that possible?

Edit: I found this in the change log

version 1.7.7
- Chinese Traditional, thanks @old2tan
- Chinese Simplified, thanks @everdream
- Browse by folder: also sort by Date Added
- Update addon.xml

Does this mean you already fixed this? I can't find the option to sort. The only option there is is to change the order (ascending or descending) but the list doesn't respond to that. It stays in ascending order.
